tests/modules/mlstudio/cdk/test_sagemaker_studio_stack.py

```diff
@@ -112,3 +114,32 @@ def test_resources_sgmstudio_extension_stack_created(db, env_fixture):
     # Assert that we have created a SageMaker domain
     # TODO: Add more assertions
     template.resource_count_is('AWS::SageMaker::Domain', 1)
+
+
+@pytest.mark.skipif(
+    os.getenv('CHECKOV_ACTIONS', 'false') != 'true', reason='Pytest used for Checkov Scan CDK Synth Output'
+)
+def test_checkov_smstudio_extension(db, env_fixture):
+    app = App()
+    stack = MockEnvironmentSageMakerExtension(
+        app,
+        'SagemakerExtension',
+        env=env_fixture,
+        db=db,
+    )
+
+    template = json.dumps(app.synth().get_stack_by_name('SagemakerExtension').template)
+    with open('checkov_smstudio_extension_synth.json', 'w') as f:
+        f.write(template)
+
+
+@pytest.mark.skipif(
+    os.getenv('CHECKOV_ACTIONS', 'false') != 'true', reason='Pytest used for Checkov Scan CDK Synth Output'
+)
```

Comment:
instead of typing this every time you can [define it once in a different file and share it](https://docs.pytest.org/en/stable/how-to/skipping.html#:~:text=You%20can%20share%20skipif%20markers%20between%20modules.%20Consider%20this%20test%20module%3A)

Reply:
good call out - added skipif decorator to `skip_conditions.py` file and imported everywhere used now